Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Abdoul BA <aba@nuxeo.com>
+ */
+
+repositoryUrl = 'https://github.com/nuxeo/jx-platform-test'
+
+properties([
+  [$class: 'GithubProjectProperty', projectUrlStr: repositoryUrl],
+  [$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60', numToKeepStr: '60', artifactNumToKeepStr: '5']],
+  disableConcurrentBuilds(),
+])
+
+pipeline {
+  agent {
+    label 'jenkins-nuxeo-platform-11'
+  }
+  stages{
+    stage ("Test") {
+     steps {
+       echo "NUXEO::PLATFORM Test"
```

Comment:
This is not related to the Nuxeo platform, maybe something more generic linked to the stage itself, like "Test simple stage" (we'll have more complex stages afterwards).

The stage name could also be "Simple stage".